transaction/src/manifest/e2e.rs

```diff
@@ -240,25 +240,25 @@ CALL_FUNCTION
             apply_address_replacements(
                 r##"
 CALL_METHOD
-    Address("${component_address}")
+    Address("component_sim1cqvgx33089ukm2pl97pv4max0x40ruvfy4lt60yvya744cvemygpmu")
```

Comment:
Why have we moved away from using `component_address` / `account_address` / `package_address` here? :)

Reply:
My bad, switching back.